olympicRecords.py

```diff
@@ -0,0 +1,41 @@
+#CSci 127 Teaching Staff
+#FALL 2021
+#A template for a program that returns olympic world records
+#Modified by:  ADD YOUR NAME HERE
+
+def worldRecord(gender, event):
+     """
+     Write a function worldRecord(), that takes two parameters: gender (string) and the event type (int). 
+     The function should return a float for the Olympic world record for the event. 
+
+     Gender can be: Men's Standard = "men", Women's Standard = "women"
+
+     If Men's Standard and event type is 100 meters, the record is 9.63 seconds.

```

Comment:
Can you change this comments to be in this format?
Men's Standard, event type 100 meters: record is 9.63 seconds.
Thanks!